at.vintagestory.VintageStory.yaml

```diff
@@ -1,6 +1,6 @@
 app-id: at.vintagestory.VintageStory
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: '23.08'
```

Comment:
You could change to 24.08.

Reply:
https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet7
Since dotnet7 is EOL, it's an archived repo. The final branch for this dependency is 23.08.
Is the game locked to require dotnet7, or can we upgrade dotnet as well?

Reply:
I'm pretty sure dotnet7 is required and you can't just switch to v8

Reply:
Then 23.08 is the best we can do

Reply:
I guess we'll have to wait for the maintainer to merge the changes.